python_modules/libraries/dagster-aws/dagster_aws/s3/resources.py

```diff
@@ -33,6 +33,44 @@
         description="Specifies a profile to connect that session",
         is_required=False,
     ),
+    "use_ssl": Field(
+        bool,
+        description="Whether or not to use SSL. By default, SSL is used.",
+        is_required=False,
+        default_value=True,
+    ),
+    "verify": Field(
+        str,
+        description=(
+            "Whether or not to verify SSL certificates. By default SSL certificates are verified."
+            " You can also specify this argument if you want to use a different CA cert bundle than"
+            " the one used by botocore."
+        ),
+        is_required=False,
+        default_value=None,
+    ),
+    "aws_access_key_id": Field(
+        str,
```

Comment:
```suggestion
       StringSource,
```
we probably want it to be `StringSource` ([docs](https://docs.dagster.io/_apidocs/config#dagster.StringSource)) so users can pass in either the string value or the env var key.

see best practices of handling secrets: https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets#handling-secrets

Reply:
@yuhan make sense ! Fixing it soon !